app/ui/common/toggleInput.tsx

```diff
@@ -1,4 +1,7 @@
-type ToggleInputColors = 'orange' | 'blue';
+export enum ToggleInputColors {
```

Comment:
suggestion: same thing, we can reuse one enum for all the app.

Reply:
Same answer :)